charts/boundary-chart/templates/configmap.yaml

```diff
@@ -122,6 +122,7 @@ data:
       {{- end }}
     }
     {{- end }}
+    {{- end }}
```

Comment:
Thanks for submitting! I wanted to get in a slightly more substantial change for GCP - would be willing to pull in the most recent change? Thanks!

Reply:
Merged the changed. Please review now.